openweathermap/tests/tests.py

```diff
@@ -0,0 +1,5 @@
+from dataprep.connector import Connector
+
+
```

Comment:
Can you have a try to see if this runs the sanity check successfully?

```python
def test_sanity():
    Connector("./openweathermap")
```

Reply:
![image](https://user-images.githubusercontent.com/35017006/95036337-7430d200-067c-11eb-8455-8b6999b3f803.png)

I think here it says all checks have passed. I think I may not be understanding exactly what to do but does that answer your question?

Reply:
The current sanity check is a noop. It just returns success whatever you put in the configuration file. 
```python
def test_sanity():
    Connector("./openweathermap")
```
This will allow the sanity check to actually check the configuration content.